### DISCUSSION TITLE
Not is visible some additional lines in html.

### DISCUSSION THREAD

**Discussion by mjulianacarvajal** [django-wiki/django-wiki#1377] [General]
Hi everybody, 
I'm use the lib but not is visible some additional lines written down in the code's html, exactly in the template called base site.html,  ¿How I can view all lines wrote in the deploy?

![Captura de pantalla 2024-08-26 110008](https://github.com/user-attachments/assets/01beaebf-9de7-4035-a4fc-657ea5820ee6)

And ¿other cuestion, why I dont show the icons from awesomes fonts?, when this project is deployed.
![AF](https://github.com/user-attachments/assets/51260865-3b96-48fe-b844-5cff1addece6)

Somebody. ¡can help me?. Thanks a lot.

María J.

**Comment by oscarmcm** [comment]
Hi @mjulianacarvajal thank you opening this discussion.

How are you deploying the app?
If you're editing the source files and not overriding it on your project it will never show your changes.

**Reply by mjulianacarvajal** [reply]
Oscar, Thanks for your answer(and for the development of  this tool).
Before of the update my repo in the server, I wrote the addition of some lines in the main page and the footer
https://github.com/mjulianacarvajal/Wiki/blob/master/env/Lib/site-packages/wiki/templates/wiki/base_site.html. in this moment since http://198.251.74.11:8082/ is not visible but when I see the source file in the server is visible this changes added in comparison with the original templates of the library.

My repo is https://github.com/mjulianacarvajal/Wiki

Thanks,again.

María J.

**Reply by benjaoming** [reply]
Hi Maria :wave: 

When you refer to our docs here:

https://django-wiki.readthedocs.io/en/main/customization.html

...the background about how you actually override templates is part of the Django docs:

https://docs.djangoproject.com/en/5.1/howto/overriding-templates/

So when you have setup your project's template directory, you should place your custom file in `wiki/base.html` as shown in our docs.
